### PR TITLE
[T2A4][T16-A4] Andrew Sugiarto

### PR DESCRIPTION
**Pull request by Azraelos**

### CODE REVIEW

**Comment by Azraelos** [comment]
Ready for review.

File: src/seedu/addressbook/AddressBook.java [hunk 1]
```diff
@@ -484,15 +484,30 @@ private static String getMessageForPersonsDisplayedSummary(ArrayList<String[]> p
      */
     private static ArrayList<String[]> getPersonsWithNameContainingAnyKeyword(Collection<String> keywords) {
         final ArrayList<String[]> matchedPersons = new ArrayList<>();
+        final ArrayList<String> keywordsInLowerCase = changeWordsToLowercase(keywords);
```
Comment (sooyj) [inline src/seedu/addressbook/AddressBook.java:487]: Good code modification. Variable names are meaningful with clear code logic.

File: src/seedu/addressbook/AddressBook.java [hunk 2]
```diff
@@ -484,15 +484,30 @@ private static String getMessageForPersonsDisplayedSummary(ArrayList<String[]> p
      */
     private static ArrayList<String[]> getPersonsWithNameContainingAnyKeyword(Collection<String> keywords) {
         final ArrayList<String[]> matchedPersons = new ArrayList<>();
+        final ArrayList<String> keywordsInLowerCase = changeWordsToLowercase(keywords);        
         for (String[] person : getAllPersonsInAddressBook()) {
-            final Set<String> wordsInName = new HashSet<>(splitByWhitespace(getNameFromPerson(person)));
-            if (!Collections.disjoint(wordsInName, keywords)) {
+            final Set<String> wordsInName = new HashSet<>(changeWordsToLowercase(splitByWhitespace(getNameFromPerson(person))));
```
Comment (sooyj) [inline src/seedu/addressbook/AddressBook.java:489]: Good reuse of the method changeWordsToLowercase().

**Review by sooyj — COMMENTED** [review]
Reasonable attempt.

Note that you missed:
- Update the method comment
- Update the command help message

**Pull request closed without merging**